go.mod

```diff
@@ -48,4 +46,5 @@ require (
 	golang.org/x/sync v0.8.0 // indirect
 	golang.org/x/sys v0.23.0 // indirect
 	golang.org/x/text v0.17.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
this is coming from

```
o mod why gopkg.in/yaml.v3
# gopkg.in/yaml.v3
k8s.io/kube-openapi/cmd/openapi2smd
github.com/google/gnostic-models/openapiv2
gopkg.in/yaml.v3
```

Reply:
I See, and the type is required, breaking this PR 😢 
```
Error: pkg/util/proto/document_v3.go:291:31: cannot use s.GetDefault().ToRawInfo() (value of type *"gopkg.in/yaml.v3".Node) as *"sigs.k8s.io/yaml/goyaml.v3".Node value in argument to parseV3Interface

```

Reply:
@jpbetz should we move `github.com/google/gnostic-models/openapiv2` to our fork too?

Reply:
@Jefftree tried to mean, move the yaml imported version, not the package :)